readthedocs/templates/base.html

```diff
@@ -151,19 +151,19 @@ <h4>{% trans 'Get Involved' %}</h4>
 
           <ul>
             <li>
-              <a href="https://docs.readthedocs.io/en/latest/getting_started.html">{% trans 'Getting Started Guide' %}</a>
+              <a href="https://docs.readthedocs.io/page/getting_started.html">{% trans 'Getting Started Guide' %}</a>
```

Comment:
Now this page redirects to `/intro/getting-started-with-sphinx.html`

Reply:
I have pushed the changes.
Thanks.